web3-utils/truffle-config.js

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
Again not sure on why we use truffle.

Can't we just compile with `solc`? And then pass the compiled contract to the call that deploys it? 
I would like to simplify this and not require `ganache` or `truffle` if they're not strictly required.

Reply:
I replaced with solc!